docs/inflight_exit_scenarios.md

```diff
@@ -0,0 +1,409 @@
+# Simple In-flight Exit
+
+Alice sends tokens to Bob in transaction `tx` which has one input and one output. The block is withheld. 
+
+Bob calls `watcher/status` and gets a response:
+
+```json
+{
+    "version": "1",
+    "success": true,
+    "data": {
+        "last_validated_child_block_number": 10000,
+        "last_mined_child_block_timestamp": 1535031020,
+        "last_mined_child_block_number": 11000,
+        "eth_syncing": true,
+        "byzantine_events": [
+            {
+                "event": "block_withholding",
+                "details": {
+                    "blockhash"  : "DB32876CC6F...",
+                    "blocknum"  : 10000,
+                }
+            }
+        ],
+        "inflight_txs": [
+            {
+                "txhash": "230C450180808080...",
+                "txbytes": "F3170101C0940000...",
+                "input_addresses": [
+                    "0x1234..."
+                ],
+                "output_addresses": [
+                    "0x7890..."
+                ],
+            }
+        ]
+    }
+}
+ ```
+He notices that the chain is byzantine and there is an in-flight transaction with his address as an output.
+
+Bob starts an in-flight exit.
+
+#### 1. Get the exit data
+`/inflight_exit.get_data` 
+```json
+ {
+    "txbytes": "F3170101C0940000..."
+ }
+ ```
+
+ response:
+```json
+ {
+    "data": {
+        "txbytes": "F3170101C0940000...",
+        "sigs": "7C29FB8327F60BBFC62...",
+        "input_txs" : [
+            "F81891018080808...",
+            "2A0341808602A01..."
+        ],
+        "input_proofs" : [
+             "CEDB8B31D1E4C...",
+             "A67131D1E904C..."
+        ]
+    }
+ }
+ ```
+
+#### 2. Start the IFE
+```
+RootChain.startInFlightExit(
+    response.data.txbytes, 
+    response.data.input_txs, 
+    response.data.input_proofs, 
+    response.data.sigs, 
+    {"value": inFlightExitBond}
+)
+```
+
+#### 3. Piggyback his output
```

Comment:
the user sort of knows their "transaction" and their "exposure" in that transaction, but wouldn't it be more "by the books" to first wait on an appropriate information about piggybacking opportunity, ask for piggyback input and piggyback? 

Except for document clarity it is also reasonable, since then the user is certain that the exit start has mined on ethereum

Reply:
You're right - correct behaviour would be to wait for the `piggyback_available` event.